gradle/itest.gradle.kts

```diff
@@ -9,10 +9,20 @@ sourceSets {
     }
 }
 
+val copySqlSchema = tasks.create<Copy>("copySqlSchema") {
+    from("$rootDir/infrastructure/src/main/resources/setup/database/Schema.sql")
```

Comment:
one thing here is that we need slightly different files for MySQL in mongo mode and MySQL in normal mode. We may fix this here or in another ticket.